### PR TITLE
audio and video recorders options moved to plugin administration

### PR DESCRIPTION
**Pull request by pitrackster**

### CODE REVIEW

**Comment by pitrackster** [comment]
docker link : http://pr-879-1469778226.travis-build.claroline.net/

File: plugin/audio-recorder/Controller/AudioRecorderController.php [hunk 1]
```diff
@@ -26,6 +27,24 @@ public function __construct(AudioRecorderManager $manager)
     }
 
     /**
+     * @Route(
+     *     "/configure/form",
+     *     name="audio_recorder_config_form"
+     * )
+     * @ParamConverter("authenticatedUser", options={"authenticatedUser" = true})
```
Comment (Elorfin) [inline plugin/audio-recorder/Controller/AudioRecorderController.php]: Prefer the use of `@EXT\ParamConverter("user", converter="current_user")`. The converter you use is bloated and will be removed

**Comment by Elorfin** [comment]
LGTM

**Comment by kitan1982** [comment]
LGTM

**Pull request merged**